Dockerfile.ci-rp

```diff
@@ -25,20 +43,21 @@ COPY cmd cmd
 COPY pkg pkg
 COPY test test
 
-# lint
-RUN golangci-lint run --verbose
+# Ensure JS assets are available before generating Go code
+COPY --from=portal-build /build/pkg/portal/assets/v2/ /app/pkg/portal/assets/v2/build
```

Comment:
bug: still have an invalid path here, this ends up creating `/app/pkg/portal/assets/v2/build/build` directory in the rp-builder image stage.

I think we want

```
COPY --from=portal-build /build/pkg/portal/assets/v2/build /app/pkg/portal/assets/v2/build
```

Reply:
Fix the path